### DISCUSSION TITLE
Q-btn on Q-img not clickable

### DISCUSSION THREAD

**Discussion by cyberkid042** [quasarframework/quasar#11387] [General - Components / Directives / etc] [answered]
I have a q-btn added to a q-img, but the button is not clickable. 
Is there a way around this?

`

    <q-img
         src="https://images.unsplash.com/photo-1593642632559-0c6d3fc62b89ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1469&q=80" 
         :fit="cover" 
         class="hero rounded-borders">

        <q-btn
          class="q-ml-xl mobile-btn"
          color="purple-10"
          label="See All Posts"
        />

     </q-img>
`

Thanks.

**Comment by rstoenescu** [comment, marked as answer]
Add "all-pointer-events" Quasar helper CSS class on the QBtn.
As exemplified in https://quasar.dev/vue-components/img#example--native-context-menu (last example with a QIcon)

**Reply by cyberkid042** [reply]
Glad to get a direct help from you. Keep up the good help with quasar. I know you are improving this and adding more features to it. Keep it up!
